docs/metrics.md

```diff
@@ -127,3 +127,17 @@ Count*. The count metrics are illustrated by the following examples:
   the server. *Request Count* = 2, *Inference Count* = 9, *Execution
   Count* = 1.
 
+## Custom Metrics
+
+Triton exposes a C API to allow users and backends to register and collect
+custom metrics with the existing Triton metrics endpoint. It is up to the user
+to manage the lifetime of their custom metrics via the C API, Triton will not
+handle it for you.
+
+The 
+[identity_backend](https://github.com/triton-inference-server/identity_backend)
```

Comment:
Link to the specific section in readme for the backend once you add the behavior. In the identity backend readme too, link back to the Custom Metrics here.